.jules/quality/runs/2025-02-19.md

```
@@ -17,3 +17,21 @@ Target: `crates/tokmd-config`
 ## Outcome
 - `UserConfig` is now deterministic.
 - Added regression test `tests/determinism.rs`.
+
+---
+
+## Context (Run 2)
```

markdownlint-cli2 (0.20.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)